Traceback (most recent call last):
  File "server.py", line 109, in <module>
    site = server.Site(hatena.Setup())
  File "/home/pi/hatena/hatena.py", line 198, in Setup
    root = Root()
  File "/home/pi/hatena/hatena.py", line 50, in __init__
    self.dsResource = ds()
  File "/home/pi/hatena/hatena.py", line 81, in __init__
    self.region = UgoRoot()
  File "/home/pi/hatena/hatena.py", line 97, in __init__
    LoadHatenadirStructure(self)
  File "/home/pi/hatena/hatena.py", line 193, in LoadHatenadirStructure
    LoadHatenadirStructure(folder, os.path.join(path, foldername))
  File "/home/pi/hatena/hatena.py", line 174, in LoadHatenadirStructure
    Resource.putChild(filename[:-3], UGOXMLResource(os.path.join(path, filename)))
  File "/home/pi/hatena/hatena.py", line 148, in __init__
    self.ugofile = self.ugofile.Pack()
AttributeError: 'bool' object has no attribute 'Pack'
Traceback (most recent call last):
  File "server.py", line 109, in <module>
    site = server.Site(hatena.Setup())
  File "/home/pi/hatena/hatena.py", line 198, in Setup
    root = Root()
  File "/home/pi/hatena/hatena.py", line 50, in __init__
    self.dsResource = ds()
  File "/home/pi/hatena/hatena.py", line 81, in __init__
    self.region = UgoRoot()
  File "/home/pi/hatena/hatena.py", line 97, in __init__
    LoadHatenadirStructure(self)
  File "/home/pi/hatena/hatena.py", line 193, in LoadHatenadirStructure
    LoadHatenadirStructure(folder, os.path.join(path, foldername))
  File "/home/pi/hatena/hatena.py", line 174, in LoadHatenadirStructure
    Resource.putChild(filename[:-3], UGOXMLResource(os.path.join(path, filename)))
  File "/home/pi/hatena/hatena.py", line 147, in __init__
    self.ugofile = UGO().ReadXML(filepath, False)
  File "/home/pi/hatena/Hatenatools/UGO.py", line 387, in ReadXML
    ugo_xml = ET.parse(xmlfile).getroot()
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1182, in parse
    tree.parse(source, parser)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 656, in parse
    parser.feed(data)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1642, in feed
    self._raiseerror(v)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1506, in _raiseerror
    raise err
xml.etree.ElementTree.ParseError: mismatched tag: line 6, column 3
